modules/mobile/spec/support/iam_session_helper.rb

```diff
@@ -16,6 +16,12 @@ def iam_headers(additional_headers = nil)
     headers
   end
 
+  def iam_headers_no_camel(additional_headers = nil)
+    headers = { 'Authorization' => "Bearer #{access_token}" }
+    headers.merge!(additional_headers) if additional_headers
+    headers
+  end
```

Comment:
I wanted to make this a parameter on the original method but would have required so many changes to existing specs that it was easier to make a new method.